### PR TITLE
[skip ci] Breaking up VSAN to two tests

### PR DESCRIPTION
**Pull request by rajanashok**
Breaking the VSAN to two different robot tests-simple and complex in order to get around the 1 hour drone timeout.

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
lgtm

File: tests/manual-test-cases/Group5-Functional-Tests/5-6-VSAN-Complex.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Test 5-6 - VSAN
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-6-VSAN-Complex.md]: VSAN-Complex

File: tests/manual-test-cases/Group5-Functional-Tests/5-6-VSAN-Complex.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#References:
+[1 - VMware Virtual SAN](http://www.vmware.com/products/virtual-san.html)
+
+#Environment:
+This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
+
+#Test Steps:
+1. Deploy a new vCenter in Nimbus:  
+```--testbedName test-vpx-4esx-virtual-fullInstall-vcva-8gbmem```
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-6-VSAN-Complex.md]: complex test bed

File: tests/manual-test-cases/Group5-Functional-Tests/5-6-2-VSAN-Complex.robot [hunk 3]
```diff
@@ -1,47 +1,9 @@
 *** Settings ***
-Documentation  Test 5-6 - VSAN
+Documentation  Test 5-6 - VSAN - Complex
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-6-2-VSAN-Complex.robot]: 5-6-2

File: tests/manual-test-cases/Group5-Functional-Tests/5-6-1-VSAN-Simple.robot [hunk 4]
```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation  Test 5-6 - VSAN
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-6-1-VSAN-Simple.robot]: and in the md file as well.

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by rajanashok** [comment]
.

**Comment by rajanashok** [comment]
.

**Pull request merged**